packages/x-charts/src/models/axis.ts

```diff
@@ -61,15 +61,32 @@ export interface ChartsAxisProps extends TickParams {
   /**
    * The font size of the axis ticks text.
    * @default 12
+   * @deprecated Consider using  `tickLabelStyle.fontSize` instead.
    */
   tickFontSize?: number;
+  /**
+   * The style applied to ticks text.
+   */
+  tickLabelStyle?: ChartsTextProps['style'];
+  /**
+   * The style applied to the axis label.
+   */
+  labelStyle?: ChartsTextProps['style'];
+  /**
+   * Defines which ticks get its label displayed. Its value can be:
+   * - 'auto' In such case, labels are displayed if they do not overlap with the previous one.
+   * - a filtering function of the form (value, index) => boolean. Warning: the index is tick index, not data ones.
+   * @default 'auto'
+   */
+  tickLabelInterval?: 'auto' | ((value: any, index: number) => boolean);
   /**
    * The label of the axis.
    */
   label?: string;
   /**
    * The font size of the axis label.
    * @default 14
+   * @deprecated Consider using  `labelStyle.fontSize` instead.
```

Comment:
Nit: Useless empty space.
```suggestion
   * @deprecated Consider using `labelStyle.fontSize` instead.
```